pkg/codegen/jenny_go_resources.go

```diff
@@ -46,6 +46,7 @@ func (ag *ResourceGoTypesJenny) Generate(kind kindsys.Kind) (*codejen.File, erro
 	if err := tmpls.Lookup("core_resource.tmpl").Execute(buf, tvars_resource{
 		PackageName:      mname,
 		KindName:         kind.Props().Common().Name,
+		Version:          kind.Lineage().First().Version().String(),
```

Comment:
Is the `First` lineage the one you want here, or the `Latest`?

Reply:
dooh -- yup.  Actually looks like a variable with the current schema is already in scope